recipes/r-sigQC/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = '0.1.14' %}
+
+package:
+  name: r-sigqc
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: sigQC_0.1.14.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/sigQC_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/sigQC/sigQC_{{ version }}.tar.gz
+  sha256: 90678a5e52eb0813efaf32f6c225fef7c73fe5ddbcf8dadd5be321bbe6756f9f
+
+build:
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - bioconductor-complexheatmap
+    - r-kernsmooth
+    - r-mass
+    - bioconductor-rankprod
+    - r-biclust
+    - r-class
+    - r-cluster
+    - r-fmsb
+    - r-gplots
+    - r-gridgraphics
+    - r-lattice
+    - r-mclust
+    - r-moments
+    - r-nnet
+    - gcc  # [linux]
+    - llvm # [osx]
+
+  run:
+    - r-base
+    - bioconductor-complexheatmap
+    - r-kernsmooth
+    - r-mass
+    - bioconductor-rankprod
+    - r-biclust
+    - r-class
+    - r-cluster
+    - r-fmsb
+    - r-gplots
+    - r-gridgraphics
+    - r-lattice
+    - r-mclust
+    - r-moments
+    - r-nnet
+
+test:
+  commands:
+    - $R -e "library('sigQC')"           # [not win]
+
+about:
+  home: https://CRAN.R-project.org/package=sigQC
+  license: file LICENSE (Restricts use)
```

Comment:
Can you check this manually? It looks wired and maybe we are not allowed to redistribute it :(

Reply:
@bgruening should be ok -- "free for non-commercial users. Non-academic users MUST have requested from the author." (I am the author)

Thanks!

Reply:
The point is the user that uses the conda package, probably don't know about this license and therefor non-academic users might not see this.